src/werkzeug/http.py

```diff
@@ -568,9 +569,11 @@ def parse_authorization_header(value):
     except ValueError:
         return
     if auth_type == b"basic":
+        from binascii import Error
+
         try:
             username, password = base64.b64decode(auth_info).split(b":", 1)
-        except Exception:
+        except Error:
```

Comment:
HTTP parsing needs to be tolerant of bad data sent by the client. Rather than failing and returning a 500 error, we choose to treat most data errors as if the value wasn't sent. `binascii.Error` only catches some of the errors that `b64decode` can raise.